sul.yaml

```
@@ -1 +1,13 @@
-alerts: []
+alerts:
+  - from: '2020-12-16T06:00:00-07:00'
+    to: '2021-01-02T22:00:00-07:00'
+    application_name: 'MyLibrary'
+    html: The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure period (December 18, 2021 - January 2, 2022). More information about <a href="https://library-hours.stanford.edu/">hours</a>
+  - from: '2020-12-16T06:00:00-07:00'
+    to: '2021-01-02T22:00:00-07:00'
+    application_name: 'SearchWorks'
+    html: The Stanford Libraries will be operating on a reduced schedule during the Stanford Winter Closure period (December 18, 2021 - January 2, 2022). More information about <a href="https://library-hours.stanford.edu/">hours</a>
+  - from: '2020-12-16T06:00:00-07:00'
+    to: '2021-01-02T22:00:00-07:00'
+    application_name: 'bento'
```

```suggestion
    application_name: 'sul_bento_app'
```

I think? If we're keeping the message consistent across applications, I think we could also collapse them to a single entry without an `application_name` filter instead?

---

@cbeer We're not doing hours this year, but I think it's still set up to receive global alerts